### DISCUSSION TITLE
epmd error for host ... timeout, if IPSEC-filter is assigned

### DISCUSSION THREAD

**Discussion by ghost** [rabbitmq/rabbitmq-server#3145] [Questions] [answered]
Hello everyone, we get a BOOT FAILED error on rabbitmq-server restart if we assign filter in IPSEC on Windows Servers (2012 R2, 2016 etc). We permitted following ports in filter properties: 4369, 5672, 5671, 15672 and 25672. All for TCP. But as soon as we activate the filter, RabbitMQ can't restart. I already searched for IPSEC-issues on the net but found nothing useful. Hope anyone can help me in any way! 
Update: seem to be (only) a timeing problem on restart of rabbitmq. If rabbitmq (epmd.exe get the host) is running, I can activate ipsec without any problems during usage of rabbitmq. Is it only a timeout-problem of epmd.exe?! Is there a possibility to expand the time for waiting for host on rabbbitmq restart?
Thanks for any help!!!
Stefan

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ does not start `epmd` on boot, the runtime does. RabbitMQ periodically starts one-off VMs to make sure `epmd` is running because on Windows it can be stopped by e.g. effective user logging out (!!). Your security settings need to allow epmd to start and bind to its port (4369) and I guess resolve its hostname. epmd is a service very limited in scope (it translates Erlang node names to range-picked ports on the current host), so it should be whitelisted for most security tools.

**Reply by dminca** [reply]
Just faced a similar issue today in a Kubernetes cluster and this discussion gave me a hint where it came from.

Turns out, I had a `NetworkPolicy` that was supposed to allow inter-node communication within a Namespace but it was blocking it instead. Cheers for the `epmd` hint 🥂
